### ISSUE TITLE
yolov8在剪枝后finetune模型时，如果设置多卡会报错，Expected all tensors to be on the same device, but found at least two devices, cuda:2 and cuda:0!

### ISSUE THREAD

**Issue by zhuangyihao1997** [VainF/Torch-Pruning#414]
请问一下，yolov8在剪枝后finetune，单卡的话正常，使用多卡报错 ， 在运行到 bbox_deltas = torch.cat((xy_centers[None] - lt, rb - xy_centers[None]), dim=2).view(bs, n_boxes, n_anchors, -1)  时报了错误  Expected all tensors to be on the same device, but found at least two devices, cuda:2 and cuda:0!

请问一下这个如何解决
